app/tests/compilers/css.js

```diff
@@ -0,0 +1,4 @@
+function noop() { return null; }
+
+require.extensions['.css'] = noop;
```

Comment:
i mean for resolving them, not for parsing.

if `some-test.js` uses `import thing from 'thing'`, then it won't find `thing.jsx` (but non-test code will).